calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/ReflectionUtils.java

```diff
@@ -50,8 +54,8 @@ static void invokeMethod(Object object, Method method, Object... args) {
         try {
             if (method == null) return;
             method.invoke(object, args);
-        } catch (IllegalAccessException | InvocationTargetException ignored) {
-            ignored.printStackTrace();
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            Log.d(TAG, "Can't invoke method using reflection", e);
```

Comment:
I'm just seen Log.w inside TypefaceUtils and do the same thing here, cause in my opinion Log is better than simple e.printStackTrace(). I can revert my change back. Anyway logging issues is a deal of separate pull request :)